packages/shared/forks/ReactFeatureFlags.testing.js

```diff
@@ -20,7 +20,6 @@ export const enableProfilerCommitHooks = __PROFILE__;
 export const enableProfilerNestedUpdatePhase = __PROFILE__;
 export const enableProfilerNestedUpdateScheduledHook = false;
 export const enableUpdaterTracking = false;
-export const enableSuspenseServerRenderer = false;
```

Comment:
Is it possible this will break tests?

Reply:
I'm guessing no since test-renderer doesn't do server rendering but it might be worth doing a test PR.

Reply:
www uses the .www feature flags file anyway where this is already enabled